### PR TITLE
revamp how were organized

### PR DESCRIPTION
**Pull request by BlakeTurner**

### CODE REVIEW

File: how-were-organized.md [hunk 1]
```diff
@@ -1,64 +1,66 @@
...
+The Data team owns our data pipelines and warehouse. They create the data infrastructure required for us to make business decisions with ever-increasing speed and accuracy, and lead data analysts across Bench in adopting better technology and practices to accomplish this. They test their data infrastructure and processes by serving the Product team directly as analysts, and by filling any gaps in reporting and analysis that emerge from requests of analysts outside the Data team.
 
-### Systems Engineering
+### Platform
 
-The Systems Engineering team is focused on improving our technology and processes to enhance the speed and quality of our engineering, with a strong focus on the reduction of technical debt. Unlike Support Engineering, they work with few distractions in order to increase the speed with which they can make improvements.
+The Platform team owns our cloud infrastructure, our build pipeline, and our development environment. They play a big role in enabling Product Engineering, IT, and Data with infrastructural support and guidance.
```
Comment (vweller) [inline how-were-organized.md:41]: The platform team own our cloud infrastructure and builds and maintains the interfaces that Benchmates use to create value for our customers. They play a big role in enabling Product Engineering, IT, and Data with infrastructural support and guidance

File: how-were-organized.md [hunk 2]
```diff
@@ -1,64 +1,66 @@
...
+### Support Engineering
 
-The IT team owns our physical infrastructure (wifi, on-prem VPN, laptops, etc) as well as the access to our software (think Okta, Salesforce, etc). They are responsible for onboarding, offboarding, and a significant amount of our security controls.
+The Support Engineering team's primary concern is KTLO in our application level, their secondary concern is supporting stakeholder requests that are "too small for the roadmap", and their tertiary concern is innovating to make both KTLO and supporting stakeholders more efficient. Support Engineering works closely with the rest of BizTech to handle the issues and outages if/when they arise. In effect, the Support Engineering team allows Product Engineering to execute with increased focus. 
 
-_NOTE: BAS actually reports into Operations, but they're a key contributor to BizTech's success_
+### Developer Experience and Impact (DXI)
 
-## Product Development
+The DXI team is focused on tools, frameworks, techniques, practices, and systems that help bring joy to the software development experience and to reduce any negative impacts, friction, and pain that our engineers experiencing while building our software.
```
Comment (vweller) [inline how-were-organized.md:53]: please include "app security" as well. 🙏

**Review by vweller — CHANGES_REQUESTED** [review]
In the image it says Security for PLA, but it is specifically the infra sec. DXI will be working with App security concerns. Can you please update the image?

**Review by vweller — APPROVED** [review]
Looks great! Thank you!

File: how-were-organized.md [hunk 3]
```diff
@@ -1,64 +1,66 @@
...
+The platform team own our cloud infrastructure including securing it. This team builds and maintains the interfaces that Benchmates use to create value for our customers, and they play a big role in enabling Product Engineering, IT, and Data with infrastructural support and guidance.
 
-### Data
+### Bench Application Support (BAS)
 
-The Data team owns our data pipelines and warehouse. They create the data infrastructure required for us to make business decisions with ever-increasing speed and accuracy, and lead data analysts across Bench in adopting better technology and practices to accomplish this. They test their data infrastructure and processes by serving the Product team directly as analysts, and by filling any gaps in reporting and analysis that emerge from requests of analysts outside the Data team.
+BAS handles all internal and external bug reports, questions, and stakeholder requests. They're also experts in our applications, and regularly provide help and training to unblock other Benchmates. Increasingly they're taking on other functions, like QA and security compliance.
```
Comment (Seanatbench) [inline how-were-organized.md:45]: I'd rewrite as:
BAS triages internal and external bug reports, questions, and stakeholder requests. They're experts in our applications, and regularly provide help and training to empower other Benchmates. Increasingly they're taking on other functions, like QA, site updates, and compliance.

File: how-were-organized.md [hunk 4]
```diff
@@ -1,64 +1,66 @@
...
+The Business Technology group exists to support the implementation, creation, and use of technology across Bench. The group's primary function is KTLO (Keeping The Lights On) and Support; indeed many of the BizTech teams use JIRA support desk to manage the support requests that they receive. It is made up of the **IT**, **Data**, **Platform**, **Bench Application Support (BAS)**, **Support Engineering**, and **Developer Experience and Impact (DXI)** teams.
 
-### Platform
+### IT
 
-The Platform team owns our cloud infrastructure, our build pipeline, and our development environment. They play a big role in enabling Product Engineering, IT, and Data with infrastructural support and guidance. 
+The IT team owns our physical infrastructure (wifi, on-prem VPN, laptops, etc) as well as the access to our software (think Okta, Salesforce, etc). They are also responsible for onboarding, offboarding, and a significant amount of our security controls.
```
Comment (b-drama) [inline how-were-organized.md:33]: The IT team owns and supports our physical infrastructure (wifi, on-prem VPN, laptops, etc) as well as the administrating and maintaining access control to various software. They are also responsible for onboarding, offboarding, and a significant amount of our security controls.

**Review by b-drama — CHANGES_REQUESTED** [review]

File: how-were-organized.md [hunk 5]
```diff
@@ -1,64 +1,66 @@
...
+The Data team owns our data pipelines and warehouse. They create the data infrastructure required for us to make business decisions with ever-increasing speed and accuracy, and lead data analysts across Bench in adopting better technology and practices to accomplish this. They test their data infrastructure and processes by serving the Product team directly as analysts, and by filling any gaps in reporting and analysis that emerge from requests of analysts outside the Data team.
 
-### Systems Engineering
+### Platform
 
-The Systems Engineering team is focused on improving our technology and processes to enhance the speed and quality of our engineering, with a strong focus on the reduction of technical debt. Unlike Support Engineering, they work with few distractions in order to increase the speed with which they can make improvements.
+The platform team own our cloud infrastructure including securing it. This team builds and maintains the interfaces that Benchmates use to create value for our customers, and they play a big role in enabling Product Engineering, IT, and Data with infrastructural support and guidance.
 
-### Data
```
Comment (Seanatbench) [inline how-were-organized.md:41]: I'd prefer to have a header for Support with BAS and Sup Eng as sub-headers beneath it. We do this in the BizTech documents to show that they're two halves of the support org. Would that formatting overcomplicate what you're trying to do here?

File: how-were-organized.md [hunk 6]
```diff
@@ -1,64 +1,66 @@
...
+BAS handles all internal and external bug reports, questions, and stakeholder requests. They're also experts in our applications, and regularly provide help and training to unblock other Benchmates. Increasingly they're taking on other functions, like QA and security compliance.
 
-### IT
+### Support Engineering
 
-The IT team owns our physical infrastructure (wifi, on-prem VPN, laptops, etc) as well as the access to our software (think Okta, Salesforce, etc). They are responsible for onboarding, offboarding, and a significant amount of our security controls.
+The Support Engineering team's primary concern is KTLO in our application level, their secondary concern is supporting stakeholder requests that are "too small for the roadmap", and their tertiary concern is innovating to make both KTLO and supporting stakeholders more efficient. Support Engineering works closely with the rest of BizTech to handle the issues and outages if/when they arise. In effect, the Support Engineering team allows Product Engineering to execute with increased focus.
```
Comment (Seanatbench) [inline how-were-organized.md:49]: Suggestion: 
Support Engineering's primary concern is KTLO in our application, balancing their remaining capacity between supporting stakeholders with features that are too small for the roadmap and innovating to make both KTLO and supporting stakeholders more efficient. Sup Eng and BAS work together to handle crises when they occur, pulling in other BizTech teams or Product Engineering if needed.

**Review by b-drama — APPROVED** [review]

**Pull request merged**